db/dcrpg/internal/stakestmts.go

```diff
@@ -198,6 +201,43 @@ const (
 			WHERE t.rnum > 1);`
 
 	// Revokes?
+
+	// Agendas
+	CreateAgendasTable = `CREATE TABLE IF NOT EXISTS agendas (
+		id SERIAL PRIMARY KEY,
+		agenda_id TEXT,
+		agenda_vote_choice TEXT,
```

Comment:
There will only ever be the three choices (yes,no,abstain).  Let's avoid `TEXT` here as the queries using CASE are likely to be slower with text.  smallint/INT2 would work.

Reply:
Noted.